tests/e2e/test_rosacli_machine_pool.go

```diff
@@ -155,3 +149,76 @@ var _ = Describe("Edit machinepool",
 
 			})
 	})
+
+var _ = Describe("Edit machinepool",
+	labels.Day2,
+	labels.FeatureMachinepool,
+	labels.NonHCPCluster,
+	func() {
+		defer GinkgoRecover()
+		var (
+			clusterID          string
+			rosaClient         *rosacli.Client
+			machinePoolService rosacli.MachinePoolService
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			clusterID = config.GetClusterID()
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			machinePoolService = rosaClient.MachinePool
+
+		})
+
+		AfterEach(func() {
+			By("Clean remaining resources")
+			err := rosaClient.CleanResources(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("will succeed - [id:38838]", func() {
+			By("Check help message")
+			output, err := machinePoolService.EditMachinePool(clusterID, "", "-h")
+			Expect(err).ToNot(HaveOccurred())
+			Expect(output.String()).Should(ContainSubstring("machinepool, machinepools, machine-pool, machine-pools"))
```

Comment:
If the output contains "machine-pools", it must contain "machine-pool", right? If so, expecting "machine-pool" is unnecessary.